seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCode.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.common.exception;
+
+public enum CommonErrorCode implements SeaTunnelErrorCode {
+    FILE_OPERATION_FAILED("COMMON-01", "File operation failed, such as (read,list,write,move,copy,sync) etc..."),
+    JSON_OPERATION_FAILED("COMMON-02", "Json covert/parse operation failed"),
+    REFLECT_CLASS_OPERATION_FAILED("COMMON-03", "Reflect class operation failed"),
+    SERIALIZE_OPERATION_FAILED("COMMON-04", "Serialize class operation failed"),
+    UNSUPPORTED_OPERATION("COMMON-05", "Unsupported operation error"),
+    ILLEGAL_ARGUMENT("COMMON-06", "Illegal argument"),
+    UNSUPPORTED_DATA_TYPE("COMMON-07", "Unsupported data type"),
+    SQL_OPERATION_FAILED("COMMON-08", "Sql operation failed, such as (execute,addBatch,close) etc..."),
+    TABLE_SCHEMA_GET_FAILED("COMMON-09", "Get table schema from upstream data failed"),
+    FLUSH_DATA_FAILED("COMMON-10", "Flush data to sink connector failed"),
```

Comment:
```suggestion
    FLUSH_DATA_FAILED("COMMON-10", "Flush data from sink connector failed"),
```

Reply:
`FLUSH_DATA_FAILED("COMMON-10", "Flush data to target failed"),` better?

Reply:
How about `flush data operation that in sink connector failed`? CC @EricJoy2048 @Hisoka-X